fw/hpack.c

```diff
@@ -3486,6 +3486,52 @@ tfw_hpack_hdr_expand(TfwHttpResp *__restrict resp, TfwStr *__restrict hdr,
 	if (!hdr)
 		return -EINVAL;
 
+	if (hdr->flags & TFW_STR_TRAILER) {
```

Comment:
I think we have `tfw_http_hdr_split` instead of all code below.

Reply:
okay, I'll have a check.

Reply:
`tfw_http_hdr_split` did a different thing. It just splits the chunks of a TfwStr into two parts and excludes `: `, but my code merges sparse chunks into two chunks, which meets the requirement of `tfw_hpack_hdr_expand`. So it cannot be replaced by `tfw_http_hdr_split`. It's tested.

Reply:
@kingluo I didn't check the code yet, but typically if a reviewer has a question on code, then any other reader is likely have the same question, so it worth commenting the code. In this case it'd be good to add a comment like you wrote above

Reply:
okay, done.